src/bp/core/routers/bots/index.ts

```diff
@@ -15,6 +15,7 @@ import AuthService, { TOKEN_AUDIENCE } from 'core/services/auth/auth-service'
 import { BotService } from 'core/services/bot-service'
 import { FlowView } from 'core/services/dialog'
 import { FlowService } from 'core/services/dialog/flow/service'
+import { FlowModification } from 'core/services/dialog/flow/typings'
```

Comment:
When adding typings, there are 3 different scenarios to determine where they should go:

1. It's only being used by this file or very few other classes, then it should be in its main file and exported
2. It's used widely across the backend, or used by the backend/admin, then it should be in `common/typings`
3. It's used by the sdk, then it should be in botpress.d.ts

We try to get only one file called typings per project (module/backend), otherwise it's a pain to find it. In your case, they could be in flow/service.ts, since it all refers to methods in that file (rename, delete, etc)